src/Components/Assets/AssetFilter.tsx

```diff
@@ -60,13 +48,8 @@ function AssetFilter(props: any) {
 
   useQuery(routes.getFacilityAssetLocation, {
     pathParams: {
-      facilityId: String(facilityId),
-      locationId: String(locationId),
-    },
-    onResponse: ({ res, data }) => {
-      if (res?.status === 200 && data) {
-        setLocation(data);
-      }
+      facility_external_id: String(facilityId),
+      external_id: String(locationId),
     },
     prefetch: !!(facilityId && locationId),
   });
```

Comment:
You can delete this `useQuery` call and avoid the unused API call entirely right? You are neither doing a setData on response, nor using the returned data from the useQuery.

Reply:
done